zuul.d/jobs.yaml

```diff
@@ -849,6 +849,8 @@
     secrets:
       - citynetwork_credentials
       - swr
+    vars:
+      go_version: '1.15'
```

Comment:
@jichenjc Its already enabled. https://github.com/theopenlab/openlab-zuul-jobs/pull/1129/files

Reply:
@ramineni why it's not running well on our CI?